Makefile

```diff
@@ -19,11 +19,11 @@ test:
 test-integration:
 	bundle exec rspec
 
-build-production: test
+build-production:
 	GOOS=linux GOARCH=amd64 $(BUILD_COMMAND) -o draupnir.linux_amd64 *.go
 
-client: test
-	GOOS=darwin GOARCH=amd64 $(BUILD_COMMAND) -o draupnir-client cli/*.go
+client:
+	GOOS=linux GOARCH=amd64 $(BUILD_COMMAND) -o draupnir-client cli/*.go
```

Comment:
This makes the client unrunnable on a mac, which is the most common dev environment. Not sure that's the most sensible thing to do - if you need a linux client do you want to just build two here, instead of one?

Reply:
Ah sorry, this was unintentional. Will change back